packages/unigraph-dev-explorer/src/components/SchemaEditor/types.ts

```diff
@@ -0,0 +1,42 @@
+export interface KeyValueField<K = any, V = any> {
```

Comment:
I'm thinking of unifying all type declarations in one place (like the json-ts declaration but we can change the name to something else)